addons/skin.estuary/xml/MusicOSD.xml

```diff
@@ -112,6 +112,30 @@
 				<ondown>noop</ondown>
 				<onleft>606</onleft>
 				<onright>600</onright>
+				<control type="radiobutton" id="62040">
+					<include content="OSDButton">
+						<param name="texture" value="osd/fullscreen/buttons/channels.png"/>
+					</include>
+					<onclick>Dialog.Close(MusicOSD)</onclick>
+					<onclick>ActivateWindow(PVROSDChannels)</onclick>
+					<visible>VideoPlayer.Content(LiveTV)</visible>
```

Comment:
referring to VideoPlayer in the MusicOSD seems a bit odd to me?
or is there any usecase i'm unaware of?

Reply:
c/p error. will change.